### ISSUE TITLE
codecov package

### ISSUE THREAD

**Issue by arnfinn** [SKDE-Felles/qmongr#3]

**Comment by arnfinn** [comment]
Should end up here: https://codecov.io/gh/SKDE-Felles/qmongr

**Comment by arnfinn** [comment]
Errors from Travis:

```
$ Rscript -e 'covr::codecov()'
Error in aggregate.data.frame(mf[1L], mf[-1L], FUN = FUN, ...) : 
  no rows to aggregate
Calls: <Anonymous> ... aggregate -> aggregate.formula -> aggregate.data.frame
Execution halted
```

**Comment by arnfinn** [comment]
Need some R code to check first...

**Comment by arnfinn** [comment]
Closed with #11
